### PR TITLE
Styling Updates and Case Insensitive Filter

### PR DESCRIPTION
**Pull request by meganft**
@neight-allen @bfpepper @vidoseaver @jbkimble 

Some small styling updates included here - images from Mark now on our landing page per discussion.
Also the filter is now case insensitive so that should be fully functional.

### CODE REVIEW

File: app/assets/javascripts/components/mentors/_body.js.jsx [hunk 1]
```diff
@@ -8,8 +8,9 @@ var Body = React.createClass({
   },
 
   searchMentors(query){
+    query = query.toLowerCase();
     let mentors = this.state.allMentors.filter((mentor) => {
-      return mentor.name.includes(query) || mentor.location.includes(query) || mentor.expertise.includes(query) || mentor.company.includes(query)
+      return mentor.name.toLowerCase().includes(query) || mentor.location.toLowerCase().includes(query) || mentor.expertise.toLowerCase().includes(query) || mentor.company.toLowerCase().includes(query) || mentor.bio.toLowerCase().includes(query)
     });
```
Comment (neight-allen) [inline app/assets/javascripts/components/mentors/_body.js.jsx]: I don't love this boolean. What if you concatenate all of the mentor properties into one big string, and check against that. Turn it into two lines.

Also, can't hurt to `toLowerCase()` on your query.

File: app/assets/javascripts/components/mentors/_index.js.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+var Index = React.createClass({
+  render() {
```
Comment (neight-allen) [inline app/assets/javascripts/components/mentors/_index.js.jsx]: Can we rename this to `MentorIndex`? We'll eventually have more than one index component.

Comment (meganft) [inline app/assets/javascripts/components/mentors/_index.js.jsx]: Yep! @neight-allen  It's now within a subfolder for 'mentors' but i'll change it to be MentorIndex

File: app/assets/javascripts/components/mentors/_body.js.jsx (hunk 1 above)
Comment (meganft) [inline app/assets/javascripts/components/mentors/_body.js.jsx]: Ok I can work on that change. I do the `toLowerCase()` on the query in line 11!

**Comment by meganft** [comment]
@neight-allen Made the changes you suggested!

**Comment by neight-allen** [comment]
Great! Thanks!

**Pull request merged**